static/js/compose.js

```diff
@@ -56,6 +56,16 @@ function autosize_textarea() {
 
 // Show the compose box.
 function show_box(tabname, focus_area, opts) {
+    var stream = stream_data.get_sub(opts.stream);
+    var can_write = _.find(stream.permissions, function (permission){
+        return permission === "can_write";
+    });
+    if (can_write === undefined && tabname === "stream") {
```

Comment:
`_.find` returns `undefined` if no value passes the test. http://underscorejs.org/#find